src/events/polygon.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  onClick: 'click',
+  onDblClick: 'dblclick',
+  onDrag: 'drag',
+  onDragEnd: 'dragend',
+  onDragStart: 'dragstart',
+  onMouseDown: 'mousedown',
+  onMouseMove: 'mousemove',
+  onMouseOut: 'mouseout',
+  onMouseOver: 'mouseover',
+  onMouseUp: 'mouseup',
+  onRightClick: 'rightclick',
+}; // note polygon events are identical to polyline events
```

Comment:
No need this comment, I prefer to keep the separate (although they are the same) to avoid coupling in case they change.

Reply:
Avoiding coupling does sound good. Thanks.